backend/src/services/kanban/Linear/Linear.ts

```diff
@@ -190,4 +191,11 @@ export class Linear implements Kanban {
       ...stateChanged,
     ];
   }
+
+  public async getSprintStatistics(
+    model: StoryPointsAndCycleTimeRequest,
+    cards: Cards
+  ): Promise<SprintStatistics> {
+    return new SprintStatistics();
```

Comment:
Can we throw an error here? like 'Linear sprint statistics service not available' or some other message.